themes/ddbasic/sass/base/form.scss

```diff
@@ -0,0 +1,306 @@
+  @include placeholder {
+    @include font('base');
+    line-height: 1em;
+  }
+  @include font('base');
+  width: 100%;
+  height: $element-height;
+  padding: 23px 15px 18px;
+  border: none;
+  border-radius: $round-corner;
+  line-height: 1em;
+  &:focus {
+    outline: none;
+  }
+}
+
+// ==========================================================================
+// Textarea
+// ==========================================================================
+
+// Textarea
+//
+// Markup:
+// <div class="resizable-textarea">
+// <textarea></textarea>
+// <div class="grippie"></div>
+// </div>
+//
+// Styleguide 1.2.2
+
+textarea {
+  @include font('base');
+  padding: 23px 15px 18px;
+  border: none;
+  border-radius: $round-corner;
+  background-color: $white;
+}
+.resizable-textarea .grippie {
+  background-color: $grey;
+  border: none;
+}
+
+
+// ==========================================================================
+// Checkbox and radiobutton
+// ==========================================================================
+
+input[type="checkbox"],
+input[type="radio"]  {
+  display: none;
+  & + label {
+    display: block;
+    padding: 7px 0 1px 42px;
+    position: relative;
+    &:before {
+      content: '';
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: 28px;
+      height: 28px;
+      margin: 0 10px 5px 0;
+      background-color: $grey;
+    }
+  }
+}
+.form-checkboxes,
+.form-radios {
+  .form-item {
+    margin-top: 0;
+    margin-bottom: 15px;
+    width: 100%;
+    float: left;
+  }
+}
+
+// Checkbox
+//
+// :checked   - When checkbox is checked
+//
+// Markup:
+// <input type="checkbox" class="{{modifier_class}}">  
+// <label>Label</label>
+//
+// Styleguide 1.2.3
+
+input[type="checkbox"] {
+  & + label {
+    &:before {
+      border-radius: $round-corner;
+    }
+  }
+  &:checked {
+    & + label {
+      @include place-icon(check, $color-text-on-primary, 28px);
+      &:after {
+        left: 0;
+      }
+      &:before {
+        background-color: $color-primary;
+      }
+    }
+  }
+}
+
+// Radiobutton
+//
+// :checked   - When radiobutton is selected
+//
+// Markup:
+// <input type="radio" class="{{modifier_class}}"> 
+// <label>Label</label>
+//
+// Styleguide 1.2.4
+
+
+input[type="radio"]  {
+  & + label {
+    &:before {
+      border-radius: 50%;
+    }
+    &:after {
+      content: '';
+      width: 10px;
+      height: 10px;
+      border-radius: 50%;
+      position: absolute;
+      top: 9px;
+      left: 9px;
+      background-color: $white;
+    }
+  }
+  &:checked {
+    & + label {
+      &:after {
+        background-color: $color-primary;
+      }
+    }
+  }
+}
+
+// ==========================================================================
+// Select list
+// ==========================================================================
+
+// Select list
+//
+// A select list needs to be wrapped in a div with the class "form-type-select" to be styled correct
```

Comment:
Add a link to or description of the purpose of this approach on select forms. Why, is the wrapper needed? Why is the width of select 110%, why is the wrapper overflow hidden, and why is the select positioned absolutely. It all seems very specific.